styles/mobile.css

```diff
@@ -1,58 +1,72 @@
-@media screen and (max-width: 768px)
-       {
-           footer hr
-           {
-               display: none;}

```

Comment:
I did quite a bit of formatting in your CSS.   Please try to work at indenting your code properly as it will help you and others to read your code